### PR TITLE
add link to GH issue label search for "area/blog"

### LINKED ISSUE #3565: [Writing Program] Add link to GH issue label search for "area/blog"
#### Current Behavior
The [Writing Program page](https://layer5.io/community/handbook/writing-program) doesn't have a hyperlink for quick reference of GH issues that have been created in search of volunteers to author resource articles and blog posts.

#### Desired Behavior
Add this hyperlink to the Writing Program page - https://github.com/layer5io/layer5/labels/area%2Fblog

---
<h3> Contributor Resources and <a href="https://layer5.io/community/handbook">Handbook</a></h3>

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- 📚 See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- 🎨 Wireframes and designs for Layer5 site in [Figma](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs). 
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](http://slack.layer5.io)

### PR DESCRIPTION
**Pull request by anthonyhamilton123**
**Description**

This PR fixes #3565

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by Nikhil-Ladha** [comment]
Please follow the contributing guide and update the PR title and template accordingly.

**Comment by leecalcote** [comment]
Thanks, @anthonyhamilton123 👍

**Comment by l5io** [comment]
🚀 Preview for commit b79acbf271545ee71dd18e01b4f18ceadd916793 at: https://63b504d1a21105042f3d8107--layer5.netlify.app

**Comment by leecalcote** [comment]
@deepeshaburse no review / update?

**Comment by l5io** [comment]
🚀 Preview for commit feaa1bb661190ce5de2dc1374d7390e116607f06 at: https://63b8a477cc26af04f9da66c2--layer5.netlify.app

**Comment by deepeshaburse** [comment]
@anthonyhamilton123, please edit the link to include all issues having that tag from all orgs (not just layer5io).

File: src/sections/Community/Handbook/writing-program.js [hunk 1]
```diff
@@ -194,6 +194,7 @@ const Writers = () => {
             <ul>
               <li><a href="https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#adding-a-blog-post">Contributing to Layer5 Blogs</a></li>
               <li><a href="https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#adding-a-resource">Contributing to Layer5 Resources</a></li>
+              <li><a href="https://github.com/layer5io/layer5/labels/area%2Fblog">View open Layer5 Github Issues</a></li>
```
Comment (Nikhil-Ladha) [inline src/sections/Community/Handbook/writing-program.js:197]: Better:
`View open Layer5 writing program issues`

Comment (leecalcote) [inline src/sections/Community/Handbook/writing-program.js:197]:
```suggestion
              <li><a href="https://github.com/layer5io/layer5/labels/area%2Fblog">View open Writing Program issues</a></li>
```

I am incorporating @Nikhil-Ladha's feedback.

**Review by leecalcote — APPROVED** [review]
Thank you, @anthonyhamilton123 👍

**Pull request merged**